### PR TITLE
call api and store locally and destructure the data

### PR DESCRIPTION
**Pull request by ashwinkhatiwada**
# Summary of change

* get the data from api
* data already stored locally
* if api limit exceeds read local data else call the api
* destructure the data

## Type of change

- [x] New feature

# How Has This Been Tested?

- [x] Manual testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 
 const riotApi = process.env.API_URL;
 const baseUrl = "https://eu.api.riotgames.com/val/content/v1/contents"
 
+let datas;
 const getData = async () => {
     const url = `${baseUrl}?api_key=${riotApi}`;
-
     try {
         const response = await axios.get(url)
```
Comment (saras-sth) [inline index.js:13]: const response = await axios.get("apilink");

    if (res.status === 429){
      ...code
    }
    if (response.status === 200) {
      console.log(response.data);
    }

Try doing something like this, as code will be much cleaner with async/await than using '.then'

File: index.js [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
             })
     } catch (error) {
         console.error
-
     }
-
-
-
 }
+await getData();
```
Comment (saras-sth) [inline index.js]: Please check on this as await is only used inside async functions, so i don't think it is required, but i could be wrong

Comment (ashwinkhatiwada) [reply index.js]: we can use await to call async function. But here it doesn't makes sense. 
Thanks for the suggestion.
await removed.

Comment (ashwinkhatiwada) [reply index.js]: I am having error if not using await. 
As the line below calling function is dependent on the data from the function. The data is undefined. And using await helped me  solve this issue.
it is behaving asynchronous, the line below is executing before the function is fully executed.

Comment (ashwinkhatiwada) [reply index.js]:
```
const data = await getData();
```

**Review by saras-sth — APPROVED** [review]

**Pull request merged**